query-graphs-js/src/querygraph.ts

```diff
@@ -211,10 +203,8 @@ export class Edge {
     // edge freely). Which means we need to clone any existing conditions (so we can modify them), and need
     // to re-freeze the result afterwards.
```

Comment:
This comment needs an update (or maybe removal). I looked for other mentions and this fortunately seems to be the last one.

Reply:
Good catch. Removed.